docs/src/main/asciidoc/mp/openapi/openapi-generator.adoc

```diff
@@ -24,8 +24,33 @@
 
 include::{rootdir}/includes/mp.adoc[]
 
-== Coming Soon
+:incdir: {rootdir}/includes/openapi
+:gen-inc: {incdir}/openapi-generator.adoc
+:helidon-client-xref: {restclient-page}
 
-The support for link:{openapi-generator-tool-site-url}[OpenAPI generator] is not available yet.
+include::{gen-inc}[tag=preamble]
```

Comment:
Keep it simple, just use `include::{rootdir}/includes/openapi/openapi-generator.adoc`

Reply:
Changed in three places.